Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 1,
         "Minor": 125,
```

Comment:
this might also require change. According to the new norm in must be 127{the current spring number} probably.